clrs/examples/run.py

```diff
@@ -55,7 +56,7 @@
 flags.DEFINE_integer('chunk_length', 16,
                      'Time chunk length used for training (if '
                      '`chunked_training` is True.')
-flags.DEFINE_integer('train_steps', 10000, 'Number of training iterations.')
```

Comment:
Minor comment: I would keep the flag defaults and supply your changes via the command line instead. The supplied defaults might be useful to keep when returning to the CLRS baseline.